### PR TITLE
Adapt samples to API changes in open systems simulator.

### PR DESCRIPTION
**Pull request by cgranade**

### CODE REVIEW

File: samples/characterization/process-tomography/tomography-sample.ipynb [hunk 1]
```diff
@@ -123,7 +121,18 @@
    "cell_type": "code",
    "execution_count": 5,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
```
Comment (anjbur) [inline samples/characterization/process-tomography/tomography-sample.ipynb:126]: Is it intended that there be an error in the output here?

File: samples/characterization/randomized-benchmarking/randomized-benchmarking.ipynb [hunk 2]
```diff
@@ -126,7 +123,18 @@
    "cell_type": "code",
    "execution_count": 5,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
+    {
+     "name": "stderr",
```
Comment (anjbur) [inline samples/characterization/randomized-benchmarking/randomized-benchmarking.ipynb:128]: Same question here, about if the error is intended

**Review by anjbur — APPROVED** [review]
I had a question about error output in two cells, but otherwise looks good to me!

File: samples/characterization/randomized-benchmarking/randomized-benchmarking.ipynb (hunk 2 above)
Comment (cgranade) [reply samples/characterization/randomized-benchmarking/randomized-benchmarking.ipynb:128]: Same here as well; these could be resolved by fixes to QInfer.

**Review by cgranade — COMMENTED** [review]
Thanks for your review, @anjbur!

**Pull request merged**